### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mmedum**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (kasperl) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: To Reproduce -> To reproduce (for consistency)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  name: Toitware
+  url: https://www.toitware.com/contact.html
```
Comment (kasperl) [inline .github/ISSUE_TEMPLATE/config.yml:4]: I wonder if we should change this to community.toitware.com once we have a friendly front for our public Slack offering up and running?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+2. Click on '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**SDK version (please complete the following information):**
```
Comment (rikkegammelmark) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Perhaps also console version

**Review by rikkegammelmark — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (mmedum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (mmedum) [reply .github/ISSUE_TEMPLATE/config.yml:4]: That is a great idea!

**Pull request merged**